File: httpsnippet/src/test/resources/output/objc/nsurlsession/cookies.m
#import <Foundation/Foundation.h>

NSDictionary *headers = @{ @"Cookie": @"foo=bar;bar=baz" };

NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:[NSURL URLWithString:@"http://mockbin.com/har"]
                                                       cachePolicy:NSURLRequestUseProtocolCachePolicy
                                                   timeoutInterval:10];
[request setHTTPMethod:@"POST"];
[request setAllHTTPHeaderFields:headers];

NSURLSession *session = [NSURLSession sharedSession];
NSURLSessionDataTask *dataTask = [session dataTaskWithRequest:request
                                            completionHandler:^(NSData *data, NSURLResponse *response, NSError *error) {
                                              if (error) {
                                                NSLog(@"%@", error);
                                              } else {
                                                NSHTTPURLResponse *httpResponse = (NSHTTPURLResponse *) response;
                                                NSLog(@"%@", httpResponse);
                                              }
                                            }];
[dataTask resume];

File: httpsnippet/src/test/resources/output/javascript/fetch/cookies.js
const fetchOptions = {
  "mode": "cors",
  "method": "POST",
  "headers": {
    "Cookie": "foo=bar; bar=baz"
  }
}

fetch("http://mockbin.com/har", fetchOptions)
 .then(response => response.json())
 .then(data => console.log(data))
 .catch(error => console.log(error));